cluster/gce/gci/configure-helper.sh

```diff
@@ -1985,6 +2008,7 @@ KUBE_SCHEDULER_TOKEN=$(dd if=/dev/urandom bs=128 count=1 2>/dev/null | base64 |
 
 setup-os-params
 config-ip-firewall
+config-sysctls
```

Comment:
@dnardo @MrHohn  By putting it here, we'll also config these sysctls on GKE master node. Are we ok with that? If not I could add some check to avoid that.

Reply:
Seems like there is an approved PR doing a similar fix: https://github.com/kubernetes/kubernetes/pull/55004.

Reply:
Don't we need `sysctl net.bridge.bridge-nf-call-iptables=1`.
We also set that in kube-proxy. And on master it's `net.bridge.bridge-nf-call-iptables=1` in the image. But we should enforce that here if we need it, right?

Reply:
I'm pretty sure kublet is already setting  net.bridge.bridge-nf-call-iptables

https://github.com/kubernetes/kubernetes/blob/4ff8cb60fa586ffb23d98bec5cb1ec892c39b2c9/pkg/kubelet/network/kubenet/kubenet_linux.go#L56

The problem here was that only the kube-proxy was setting the route_localnet sysctl, that is why you need that specifically on the master, since you are not running kube-proxy in that case.

So I'm pretty sure you don't need it.

Reply:
Also Ike is handling this in his PR already, so I think you can remove all your sysctls in favor of his:

https://github.com/ihmccreery/kubernetes/commit/c93a445aad317a60bdf3c30c7d35d4efb4e77cfd

Reply:
The problem is that we don't use `kubenet` for CRI container runtimes. But we still need to enforce this somewhere, right?

Reply:
@dchen1107 @yujuhong Should we set these sysctls in `kubelet`, `container runtime` or `cloud-init`?

Reply:
Offline discussed with @dnardo. We are moving to calico's P2P model, and we don't need the bridge sysctl for that model. Will not set this sysctl by default on each node.

I'll get rid of all the sysctl stuff in favor of ihmccreery/kubernetes@c93a445